### DISCUSSION TITLE
UI: Allow collapsing/expanding for the search results

### DISCUSSION THREAD

**Discussion by Dentrax** [community/community#9717] [Other Feature Feedback, Questions, & Ideas]
![Screen Shot 2022-01-06 at 21 32 16](https://user-images.githubusercontent.com/16493751/148432940-5ae23ee6-8219-4b93-b86b-bba096bdaa98.png)

Just similar to PR review UI: collapse/expand ability. Like the following:

![Screen Shot 2022-01-06 at 21 31 28](https://user-images.githubusercontent.com/16493751/148432834-e1565636-cde8-4c7a-8187-bb737491eac0.png)

Thoughts?

**Comment by colinwm** [comment]
What would the purpose of collapsing a search result be? Like to make a checklist, that you've looked through certain results?

**Reply by Dentrax** [reply]
It could be + to not get distracted from the unwanted search results.
